### ISSUE TITLE
Trailer / game intro video

### ISSUE THREAD

**Issue by alketii** [OpenDungeons/OpenDungeons#584]
_Edit by @akien-mga_: The video tutorial part of this issue has been adressed, but since it has been hijacked in a discussion about a trailer @eugeneloza is working on, I renamed the issue.

Hi, could someone make a video tutorial, it would be greatly appreciated.
I really would like to play the game (I like RTS games, but never played dungeon), but its very hard to get started.

**Comment by akien-mga** [comment]
That would be nice indeed. Some gameplay features are not easy to grasp and the ingame help is a good support but not exhaustive enough. Before recording a video of 0.5.0 it would be nice to have a fix for #541 though.

**Comment by Bertram25** [comment]
This is a 0.5.0 issue IMHO, while this is some kind of post release issue, IMHO.

**Comment by hwoarangmy** [comment]
Yes. When we will have the 0.5.0 RC, we can do the video. As @akien-mga said, the marked tiles bug is too visible for now. Even if the easy way to increase attenuation could hide the bug if we want for the video.

**Comment by Bertram25** [comment]
> Yes. When we will have the 0.5.0 RC, we can do the video. As @akien-mga said, the marked tiles bug is too visible for now. Even if the easy way to increase attenuation could hide the bug if we want for the video.

see #585 :D

**Comment by hwoarangmy** [comment]
Merged ^^

**Comment by hwoarangmy** [comment]
@akien-mga would you mind dealing with that ?

**Comment by eugeneloza** [comment]
Ping me if you need to voice over the tutorial or a trailer.
I'm afraid I'm not that experienced at OpenDungeons to make a quality tutorial myself, but if a script is present I may voice it after I finish "Let's play VT" (and it's gonna be soon:)).

**Comment by Bertram25** [comment]
Yeah, Eugene's voice FTW! :)

**Comment by akien-mga** [comment]
> @akien-mga would you mind dealing with that ?

I guess I could handle a basic video tutorial (i.e. I record my desktop during playing and explain the core components of the game at the same time).

As for the trailer, I don't have the skill nor the time to do video editing and stuff like that. But a trailer with @eugeneloza as an evil keeper would be so awesome :-D

**Comment by Bertram25** [comment]
Np, let's start with the video. @eugeneloza can add some voicing if we help him with the script. :)

**Comment by eugeneloza** [comment]
> I guess I could handle a basic video tutorial (i.e. I record my desktop during playing and explain the core components of the game at the same time).

It'd be rather easy for me to re-voice this tutorial. I.e. I hear what you speak (even a webcam quality will do) and repeat it.
The problem is that I'm also not skilled at video editing, so I'm voicing 'blindly' in Adobe Audition without seeing the image, just sound. So having a good time-reference would significantly increase work efficiency.

> As for the trailer, I don't have the skill nor the time to do video editing and stuff like that.

I have an interesting idea of a trailer intro... Just need some time to think it over & implement.
I've asked Samuncle from STK dev team, who makes trailers, for non-linear video editing. He pointed me to blender abilities. I think it'll be a good experience to study the possibility. And if blender catches up my Presonus Firestudio audiocard output (Linux Jack/etc don't), then I think It'll be possible.

**Comment by akien-mga** [comment]
I've recorded a basic tutorial for OD 0.5.0 with Simple Screen Recorder (awesome free software ;-)), and audacity to capture the audio. My microphone is quite bad so there is quite a bit of noise, and audacity keeps freezing on me so I haven't managed to reduce the noise.

I've done the basic editing (coupling the video + lower volume OD music + audacity recording) in kdenlive, now I'm doing the H.264/Vorbis rendering.

At first I've thought about doing a "evil keeper" tutorial that @eugeneloza could voice over, but in the end I've opted for a plain "Hello I'm Akien and here is how to play our game".

I like the voicing idea, but we'd better put such effort on a trailer IMO :)

**Comment by akien-mga** [comment]
@eugeneloza I can upload my WAV for the voice over if you want to have a look at reducing the noise.

**Comment by eugeneloza** [comment]
Cool. Give me the files (by mail or by link) and I'll do all the audio processing.

>  I've opted for a plain...

Yes. I absolutely agree that 'plain voice' is better for the tutorial.
But as for the trailer... I have a cool idea... I hope my modeling skills will be enough to XD At least that gonna be an outstanding blender experience:

a dark cave. First person view of a scared hero with a torchlight (or a flashlight - easier to make). Scared breath and heartbeat :) Suddenly an evil voice speaks something like "Not many have courage to come... Nice to see you here..." - the hero (even more scared) pulls out a sword: "I'm not afraid of you! Come on, show your face!" - "Ha-ha-ha... Which one of them?" All the distant darkness flashes up with shining eyes, evil whispers. The screen (i.e. the hero) shatters back, a few deep steps are heard, he turns back only to see a rock golem (or other cool creature) hit him to death. Darkness. "Welcome to OpenDungeons... if you are not afraid to".

I see no principal difficulties in modelling a tiny portion of a cave, some eyes. Non-realtime rendering, so the models might be not optimized. The animated rock golem model may be taken from the game. I don't think a torch/a sword model is needed, just some shimmering dull light. It'd be cool to model a flame, but not sure if I can do it.
The only problem is to make camera movement smooth, natural and nervous. I hope I'll be able to...

And then some fine frames from the game.

**Comment by hwoarangmy** [comment]
> a dark cave. First person view of a scared hero with a torchlight (or a flashlight - easier to make). Scared breath and heartbeat :) Suddenly an evil voice speaks something like "Not many have courage to come... Nice to see you here..." - the hero (even more scared) pulls out a sword: "I'm not afraid of you! Come on, show your face!" - "Ha-ha-ha... Which one of them?" All the distant darkness flashes up with shining eyes, evil whispers. The screen (i.e. the hero) shatters back, a few deep steps are heard, he turns back only to see a rock golem (or other cool creature) hit him to death. Darkness. "Welcome to OpenDungeons... if you are not afraid to".

Wow, you are preparing something much better than I thought ^^
Can't wait to see that. In any case, the script seems really nice to me

**Comment by akien-mga** [comment]
> Cool. Give me the files (by mail or by link) and I'll do all the audio processing.

Here is the WAV file for the voice over: http://remi.verschelde.fr/files/misc/od-tutorial-raw.wav
The only other file is the MKV from my screen recording with the video + OD audio, I'll just put a filter in kdenlive to lower the volume of OD's audio so that the voice is clearly audible.

**Comment by eugeneloza** [comment]
Downloaded. Signal to noise ratio is very low. I'll see what I can do without damaging the natural sound. But usually it's much better to make something like 'input mic boost'. The higher the input the less influence do the internal audiocard noises have.
I'll be done in ~20 minutes, I hope...

**Comment by Bertram25** [comment]
About the trailer: Excellent :D Hope it will turn out good.
I wish I had time to help you guys but I have unfortunately little time atm. :S

**Comment by eugeneloza** [comment]
Hmm... was playing a long time to process the audio... The noise is too strong and noise reduction artefacts showed up as reverb and swirling sound.
I'm not satisfied with the result, but I'm afraid I'm not likely to do it any better...
I made up 2 variants here:
https://www.dropbox.com/sh/58sm1t97etbdaf2/AAD4bmDFpF96rgz2ugRil4Aba?dl=0
mixdown 1 has better noise reduction with the price of less natural voice (seems fine in the mix, so I'd recommend using this one)
mixdown 2 has a more natural sound but more noise
I've tried to mix a mixdown1, it needs ~ -8 dB of music to hide the noise reduction artifacts. If you have silent parts in the video track, it'd be a good idea to silence the voice track also (if only noise is present). 

P.S. I have only 2.25 G at Dropbox, so ping me as you download the files, so that I could clear the space up.

**Comment by akien-mga** [comment]
Sorry that you had to spent so long on my poor recording. I'll definitely learn how to improve the signal to noise ratio (and maybe reduce the noise at the source altogether) for my next recording :)

I have downloaded the files, so feel free to remove them from your Dropbox. I'll check them out in a little while.

**Comment by eugeneloza** [comment]
Don't worry :). I often have to work with low quality recordings, so it's fine. I was just doing a lot of other work in parallel, so it took me longer.
I really hope that the final quality is enough for the tutorial.

**Comment by akien-mga** [comment]
Here's the first version that I made before @eugeneloza improved my voice recording: https://www.youtube.com/watch?v=NDliJOP1Z1g
I'll now work on the 2nd version and replace the first one with it.

**Comment by oyvindln** [comment]
If you send it to me when you are done, I can upload it to our official youtube channel. EDIT: Actually I'm not sure if I am able to do that, having issues accessing the account.

**Comment by eugeneloza** [comment]
Damn it!!! I love blender!!! It makes things that seemed impossible to be so simple...
https://youtu.be/IxQZc6ijaNE

**Comment by hwoarangmy** [comment]
Wow, the cave is really nice. How did you do that ?

**Comment by Bertram25** [comment]
Wow!!! @eugeneloza Mind taking me as a trainee as well? :D

**Comment by akien-mga** [comment]
Omg, how did you come up with that so fast?!

**Comment by eugeneloza** [comment]
That was actually just a cylinder, 3 copies to make 3 sleeves... and then sculpt, sculpt and sculpt with dynamic topology. Mindless sculpt. I was just scratching the walls and the floor to look homogeneous. Then... mindlessly made 3 seams and unwrapped UV and placed Nobiax (CC0) texture. And then made the camera move through it with some noise-driven shake with noise for light source also... Almost no 'intellectual' work.
And that was some good experience at blender.
Somehow youtube made the quality lower... unsharp. It looks better at computer...

**Comment by akien-mga** [comment]
@eugeneloza Better versions might come up later, Youtube often makes your video available at low res while it continues processing the high res.

**Comment by eugeneloza** [comment]
Yes... but it warned me of non-standart resolution (blender default export) - maybe this is the cause...

**Comment by akien-mga** [comment]
@oyvindln Here is the new version of the video tutorial, with @eugeneloza's mixdown n°1 (noise removed, some voice distortion, but it's better than the original version). Can you upload it on OD's channel?

_Edit:_ The link: http://remi.verschelde.fr/files/misc/od-tutorial-edited2.mkv

Title: OpenDungeons 0.5.0 Video Tutorial
Tags: some relevant stuff like "free software", "tutorial", "game development"

Description:

> OpenDungeons is a nice open source game strongly-inspired by the Dungeon Keeper series and its legacy. The game is actively developed by an international team of developers and artists, so grab it on opendungeons.github.io, give your feedback and why not get involved in future releases!
> 
> This tutorial aims at explaining the basic gameplay concepts that you need to know to play OpenDungeons 0.5.0, which lacks an in-game tutorial. Hope you won't mind the thick French accent ;-)

**Comment by oyvindln** [comment]
I have trouble accessing the account, so I can't upload it for now. (Will make a forum post later, we may have to make a new one.)

**Comment by eugeneloza** [comment]
Improved the intro concept (with normals, better resolution and more improvements): https://youtu.be/K9t_2WwXuw4

**Comment by Bertram25** [comment]
> Improved the intro concept (with normals, better resolution and more improvements): https://youtu.be/K9t_2WwXuw4

OMG, I jumped at the end of it. XD

> Edit: The link: http://remi.verschelde.fr/files/misc/od-tutorial-edited2.mkv
> @akien-mga I've watched it, it's more than perfect as a first introduction. :D 

Good stuff guys!

**Comment by akien-mga** [comment]
> I have trouble accessing the account, so I can't upload it for now. (Will make a forum post later, we may have to make a new one.)

Ok, in the meantime I uploaded the new version to my account and removed the old one: https://www.youtube.com/watch?v=P4MClQUdb0E
We can migrate it later to an official channel when we've sorted out the issues you're having with it.

**Comment by hwoarangmy** [comment]
> Improved the intro concept (with normals, better resolution and more improvements)

I didn't think you would do that so fast. You are almost there, that's amazing. IMHO, we should make more clear that the vision is from a hero (by displaying vision throught an helmet or by using a camera effect showing the hero and flying to his head).
Then, once you will have added the effects you were talking about (some creatures lurking, a sword and your big voice), it will be perfect ^^

**Comment by eugeneloza** [comment]
I didn't think it would be possible to do is so quickly either :D
I've gone through flames tutorial... It doesn't look hard to make it... So handling a torch and a sword would do. I was considering a helmet idea, but I'm not sure if I can make it 'representative' - i.e. we shall just have black areas around the screen...
I have something like this in mind https://youtu.be/m4PzNoi_KU8?t=9m1s (9 till 10 minutes).
I'll have to completely remake movement animation to (not hard after all). I was wondering, if 30 seconds are not too long for intro? Should I stay in this range? Shorter? Longer?

**Comment by akien-mga** [comment]
I changed the title to reflect the two topics of this hijacked issue ;-)
The video tutorial is done, but we can probably continue discussing the trailer here as GitHub does not provide an easy way to fork the discussion AFAIK.

Adding @Danimal696 to the discussion, because he might love @eugeneloza's previews :)

**Comment by hwoarangmy** [comment]
> I have something like this in mind https://youtu.be/m4PzNoi_KU8?t=9m1s (9 till 10 minutes).

Seems good. If we see the hand, torch and sword, it will be clear that we are following a hero. But it may be more difficult than the helmet.

**Comment by eugeneloza** [comment]
> But it may be more difficult than the helmet.

Actually sword and a torch look easier for me :) I just move 4 objects (camera, lamp, sword & torch) together, adjusting them each key frame & randomizing f-curves. No need to make them perfectly aligned and some errors in position would only add 'uncertainity' to movement. No hands, I won't be able to model them yet (I tried). 
While the helmet would require very rigorous fixation of location&rotation to the camera.
I hope I'll be able to try it today (maybe even finish it), but still have a lot of work to do (which I had to do yesterday actually :D).

The most difficult thing for me now is to find the steps&heartbeat sounds. I heard swrod&rock hit at Opengameart, and I hope I'll be able to find more needed sounds there.

**Comment by Danimal696** [comment]
eh, you bastard, you are becoming better than me ;) . Helmet migth be too much, if you still want to add it, why not use a black mask that blocks surrounding vision (no need to show metal), you can get many hand sets from blendswap:
http://www.blendswap.com/search?keywords=hand&object_class=Blend&comparator=contains&is_fan_art=1&blend_license=&blender_version=

Heart sounds can be replaced with panting and heavy panting, but would be nice to have (we need them for the Dungeon Heart anyways)

The only critic to the video is the ground being too bumpy, a nice blood curling scream and hitting sound at the end would be nice, also maybe adding small red eyes flashing at some corridor that go out once the hero looks with alert that way, making him increase his fear and panting/heart rate.
Over half a minute is a pretty good size for a teaser trailer.

**Comment by eugeneloza** [comment]
> you are becoming better than me

And of course, thanks again for teaching me and giving me the practice!!! :P

> hand sets from blendswap

Wow! Cool!!! Exactly what I needed!

> The only critic to the video is the ground being too bumpy, a nice blood curling scream and hitting sound at the end would be nice, also maybe adding small red eyes flashing at some corridor that go out once the hero looks with alert that way, making him increase his fear and panting/heart rate.
> Over half a minute is a pretty good size for a teaser trailer.

Yeah... I planned to fix&add all this. I'll make the ground more even also or make the camera follow the floor, as it looks unnatural now.

> Heart sounds can be replaced with panting and heavy panting

Planned :) I'll do voice acting soon as I finish the video part. Actually I wanted both - heartbeat and panting (as in example video from Oblivion above).

**Comment by eugeneloza** [comment]
At this point I've sketched up a timeline:
0..10 seconds - intro, the hero enters the cave
10-16 seconds - Keeper speaks welcome
16-20 seconds - Hero answers
20-24 seconds - Keeper laughs and answers
24-27 seconds - eyes & whispers
27-30 seconds - Rock golem action
Looks rather dense, no 'wasting time'. Are 10 seconds ok for intro? Seems to be atmospheric and not too long.

**Comment by akien-mga** [comment]
10 seconds are completely OK for the atmospheric intro IMO. I think you can even go higher than 30 s for the whole trailer if you want, maybe keeping it under 1 min is a good idea, but it's not rare to see 2 or 3 minutes long trailer for games.

**Comment by eugeneloza** [comment]
30 seconds it's just a game-irrelevant atmospheric intro. And then another ~1 minute of game fragments, showing&commenting game peculiarities. We'll discuss that part later.
Moreover, I'm not good at non-linear-video-editing. In Avidemux it'll take me stupidly-long amount of time + low quality result. I'll need to use something better, like blender video editor... but I have never done anything like that before. Usually I have been just using Avidemux to substitute an audio track to a one-chunk video.

**Comment by hwoarangmy** [comment]
IMHO, 10 seconds is ok for introducing the fact we are seeing trough an hero eyes and we are in the underground. For the rest, if you need some more time, between 30s and 1 minute, there is no problem...

**Comment by Bertram25** [comment]
The tutorial is out, btw: http://forum.freegamedev.net/viewtopic.php?f=36&t=6269

I'm assigning @eugeneloza for the trailer as he is simply doing it. :)

**Comment by Bertram25** [comment]
@eugeneloza I've invited you in the OD developpers Team.
This simply means you're recognized as an active contributors and can be assigned to tasks. Not much more, I'm afraid, but welcome in any way. :D 
I think nobody will have a word against it. (If you do, please speak now, or... ;])

**Comment by akien-mga** [comment]
> @eugeneloza I've invited you in the OD developpers Team.

Nice :+1:

**Comment by eugeneloza** [comment]
Thanks! I'm afraid, not too much use of me now (just trailer & voice)... but anyway, I'll try to do my best where I can :)

**Comment by hwoarangmy** [comment]
> I'm afraid, not too much use of me now (just trailer & voice)

Well, that's already good (and so far, the trailer seems pretty nice).

**Comment by Bertram25** [comment]
You've done a lot already. :)

**Comment by alketii** [comment]
Thanks @akien-mga  for the video, I learned quite a lot from it. I practiced a bit and the game play was awesome, rally nice game.

Thanks

**Comment by Bertram25** [comment]
@eugeneloza @akien-mga Would you prefer to see the trailer done before releasing? Or is it ok if we release officially, and show the trailer in a later news? IMO, both are fine, the latter will make the trailer more visible as it will have its own news.

**Comment by akien-mga** [comment]
IMHO we should release 0.5.0 and publish the trailer when it's ready. I wouldn't want to put pressure on @eugeneloza to finish the trailer asap, and 0.5.0 is ready, so let's get it out and work on the awesome 0.6.0 :-)

**Comment by Bertram25** [comment]
Yeah, agreed.

**Comment by hwoarangmy** [comment]
Yep, if we wait for everything to be ready, we will not move very fast ^^

**Comment by eugeneloza** [comment]
I might finish the trailer intro till the weekends... but about the 'main part' I can't say how fast it'll go. I've never done any serious non-linear videoediting before, actually I've used only ugly Windows Movie Maker. So, I'm rather unsure if I'll be able to find&master new software quickly.

**Comment by Danimal696** [comment]
no pressure here :) , it will be done when its done, if its takes too long we will use it for 0.6

**Comment by hwoarangmy** [comment]
> no pressure here

No pressure, no pressure... Now that I've seen the first part of the trailer, I want to see the end...

More seriously, @eugeneloza, why do you need video editing ? Isn't blender enough for what you had in mind (excepted sounds) ? From what we could see in your last video, adding some creatures lurking or some eyes doesn't seem very hard compared to what you've done (but I pretty noob with blender so I may be wrong).

**Comment by akien-mga** [comment]
> More seriously, @eugeneloza, why do you need video editing ? Isn't blender enough for what you had in mind (excepted sounds) ? From what we could see in your last video, adding some creatures lurking or some eyes doesn't seem very hard compared to what you've done (but I pretty noob with blender so I may be wrong).

I think the preview he showed us would only be the first part of the trailer. IIRC correctly afterwards there would be in-game recordings, etc., so it would require editing to put everything together.

I've used kdenlive recently to do the minimal video editing that was needing for the tutorial (I just had to cut the end of the video + add fade-in/fade-out + add the voice recording on top of it), it's relatively easy to handle (not so different to Windows Movie Maker from what I remember, but hopefully it lets you use something else than WMV...).

**Comment by eugeneloza** [comment]
I have been struggling with torch flame for a few days... not too much luck yet... maybe I go the wrong way... A nice-looking frame takes ~1 hour to render and still is not natural... hmm... I should try another flame tutorial :)

I need video editing because as I see the trailer structure:
1. intro
'Have you ever been afraid of the dark? No? Maybe the dark is afraid of you? Then you are one of us!'
2. game features (short ~5 sec frames for each game feature plus some epic battles screencast)
'Build your dungeon'
'Welcome your faithful minions'
'Explore'
'Find gold and adventure'
'And eliminate your enemies'
'Let them tremble before the Great Master'
3. some final statement
'Open Dungeons... the place where all your evil dreams come true!'

I.e. I need ~20 short screencasts of the gameplay besides the cinematic intro, merge all this together. Avidemux allows to do it, but in a very ugly way (I need pre-cut videos for the screencasts). Blender might do this, but it seems like a hard way. Youtube does this perfectly, but with lossy quality. Cinelerra might do the job, but I failed to use it to create photo-casts...

**Comment by eugeneloza** [comment]
I have been struggling with torch flame for a few days... not too much luck yet... maybe I go the wrong way... A nice-looking frame takes ~1 hour to render and still is not natural... hmm... I should try another flame tutorial :)

I need video editing because as I see the trailer structure:
1. intro
'Have you ever been afraid of the dark? No? Maybe the dark is afraid of you? Then you are one of us!'
2. game features (short ~5 sec frames for each game feature plus some epic battles screencast)
'Build your dungeon'
'Welcome your faithful minions'
'Explore'
'Find gold and adventure'
'And eliminate your enemies'
'Let them tremble before the Great Master'
3. some final statement
'Open Dungeons... the place where all your evil dreams come true!'

I.e. I need ~20 short screencasts of the gameplay besides the cinematic intro, merge all this together. Avidemux allows to do it, but in a very ugly way (I need pre-cut videos for the screencasts). Blender might do this, but it seems like a hard way. Youtube does this perfectly, but with lossy quality. Cinelerra might do the job, but I failed to use it to create photo-casts...

**Comment by Bertram25** [comment]
@eugeneloza needs quite a few things if we want it to work. :) But it's cool!

> no pressure here :) , it will be done when its done, if its takes too long we will use it for 0.6

k, I'll move this to 0.6.0 since it may take a bit of time. We should split the work to be done.
Can someone handle the raw vids needed?

> Cinelerra might do the job, but I failed to use it to create photo-casts...

@eugeneloza Without pushing at all, what about kdenlive?

**Comment by Bertram25** [comment]
Moving to later so that @eugeneloza has time for this.

**Comment by eugeneloza** [comment]
I'm in Carpathians now, we're holding a children camp :) There was tons of preparations for it, so I didn't have enough time to finish this one by now. I'll be back home at ~13 July and somewhere there I'll be able to resume the work on the Tutorial. Thanks to Pomodora technique, I hope, I'll work more efficiently on it in comparison to previous half-year.
(P.S. I finally gave up trying to make the torch - for some stupid reason it just doesn't work, and I'll try to get into moving fire modeling later. Maybe I'll add some static fire in the cave.)

**Comment by Danimal696** [comment]
Have fun with all the kiddos!

**Comment by eugeneloza** [comment]
Oh yes, we do! :D 
![aimg_9998 55](https://cloud.githubusercontent.com/assets/10726162/8510248/e56775a0-22e3-11e5-83cd-9c6090fd4b9f.jpg)

**Comment by Bertram25** [comment]
Eh eh :)

**Comment by hwoarangmy** [comment]
@eugeneloza Any news about the video ? If you cannot improve it, could you please upload it in the media repository so that we can have a look ?

**Comment by eugeneloza** [comment]
Sorry... I've been excessively lazy last time... I seem to be back in normal right now, so I'll try to finish it asap :) I'll also upload the sources tomorrow (because they are at my home computer and no Internet at home at the moment).

**Comment by hwoarangmy** [comment]
> Sorry... I've been excessively lazy last time...

Well, it is a free time project so nobody should yell at you ^^

> I seem to be back in normal right now, so I'll try to finish it asap :)

That would be great as the parts you have shown looked promising

> I'll also upload the sources tomorrow

Thanks. I have been playing with blender some time ago and I would be interested to see how you managed to do the video. I might have questions, though, as I'm not very used to this great tool.

**Comment by eugeneloza** [comment]
This is it: https://www.dropbox.com/s/q3l9nfcenkulecc/OD_cave.zip?dl=0
The working version is "4". Later - I've been struggling to make the torch, but failed. I think I should forget about it and just add a sword.

**Comment by hwoarangmy** [comment]
IMHO, you should go for the easiest way to make it work and then, focus on improving what you think can be. Of course, a torch would be cool but if it is too hard, it is not required.

For the record, I've found on open game art torches that seem to use particle effects (according to the pictures). If you want to have a look:
http://opengameart.org/content/izchaks-torches
http://opengameart.org/content/torch-with-fire

Thanks for uploading the file. Do you want to upload it on the media repo ? If you want, I can do it for you but if you don't know how to do it, just ask and we will gladly help you.

**Comment by eugeneloza** [comment]
Yeah... I've been trying to implement particles-torch, but failed. I've seen good tutorials, but I'm still to newbie in blender. That's why I've postponed the work...
And yes, I'll try to implement everything simpler and hopefully finish it this time :D. 

No, I don't know how to upload something at MediaRepo... But, if there's no haste. We'll have 4 days off in Ukraine tomorrow and I hope I'll be able to finish it by Wednesday.

**Comment by hwoarangmy** [comment]
4 days off ? What is it for ?

**Comment by eugeneloza** [comment]
We have a state holiday at 8th March (Women's day), and Monday was 'swapped' with Saturday. P.S. Sorry, I didn't make anything relative to the video yet, I have been too inspired by an overland generation algorithm for my game (those data structures nearly made my brain boil)... but eventually looks like it's going to launch up this week.

**Comment by hwoarangmy** [comment]
ah ok, it is 4 days including the week-end. I thought it was some important event that I didn't know anything about ^^
